lib/versioning/node/index.spec.ts

```diff
@@ -21,4 +21,18 @@ describe('semver.getNewValue()', () => {
       })
     ).toEqual('~8.2.0');
   });
+  it('isStable', () => {
+    const now = new Date('2020-09-01').getTime();
+    expect(isStable('16.0.0', now)).toBeFalse();
+    expect(isStable('15.0.0', now)).toBeFalse();
+    expect(isStable('14.9.0', now)).toBeFalse();
+
+    expect(isStable('12.0.3', now)).toBeTrue();
```

Comment:
Technically this was not a stable release, it was only stable once it hit LTS

Reply:
🤔 the schedule doesn't contain enough information for the minor and patch release, which has become the lts. So we can  only assume the full major is stable and propose a update to latest minor and patch.

Reply:
So, we need to start from `Maintenance LTS` instead of `Active LTS`? https://nodejs.org/en/about/releases/
If so, then I've just fixed it.

Reply:
Yes, it's unclear to me how this schedule is "sliding". From the site I assumed it's per-major. Per-minor schedule also would make sense to me, though I found nothing about this 🤔